modules/i18n/src/translations/index.ts

```diff
@@ -21,6 +21,7 @@
  */
 export * from "./en-US";
 export * from "./fr-FR";
-// export * from "./pt-BR";
+export * from "./pt-BR";
 export * from "./si-LK";
+export * from "./es-ES";
 // export * from "./ta-IN";
```

Comment:
Shall we remove this commented line, if we don't have immediate plans to support this?

Reply:
Agree. Fixed by https://github.com/wso2/identity-apps/pull/3801/commits/2176071a3b03fe1aeb3573dde3c8a8354360d6a4